project.yml

```diff
@@ -9,7 +9,7 @@ targets:
   App:
     type: application
     platform: iOS
-    deploymentTarget: 14.0
+    deploymentTarget: 14.5
```

Comment:
I did not explore atm, but this method https://developer.apple.com/documentation/callkit/cxprovider/reportnewincomingvoippushpayload(_:completion:) is available for 14.5 and newer.

Reply:
If 14.0 is what we target, I def have to dig more.

Reply:
14.5 is fine 👍